api/responses/response.controller.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { Response } = require('../../queries');
+
+exports.getAll = async (req, res) => {
```

Comment:
Need to adjust for Express I'm afraid! Errors no longer auto-collected from async!

Reply:
Mmm actually it is catching the error. I didn't set up yet an asynchronous middleware for error handling purpose, I only had this try catch block for this error.

Reply:
Heh, this is weird, somehow I didn't see the try/catch!!! Yeah cool, no worries, error handling middleware can come later of course.